.github/workflows/test-sdks-remote.yml

```diff
@@ -4,9 +4,30 @@ on:
   push:
     branches:
       - main
+      - tp/workflows/round2
   workflow_dispatch:
 
 jobs:
+
+  test-java-sdk:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Display workflow details
+      shell: bash
+      run: |
+        echo "Testing Eppo-exp/java-server-sdk"
+    - name: Run remote workflow
+      uses: convictional/trigger-workflow-and-wait@v1.6.1
+      with:
+        owner: Eppo-exp
+        repo: java-server-sdk
+        workflow_file_name: lint-test-sdk.yml
+        ref: tp/workflows/remote
```

Comment:
If you merge these first I imagine you won't need this

Reply:
that's right; will remove these refs after SDK PRs are merged